db/db_impl/db_impl_secondary.cc

```diff
@@ -17,8 +17,9 @@ namespace ROCKSDB_NAMESPACE {
 
 #ifndef ROCKSDB_LITE
 DBImplSecondary::DBImplSecondary(const DBOptions& db_options,
-                                 const std::string& dbname)
-    : DBImpl(db_options, dbname) {
+                                 const std::string& dbname,
+                                 const std::string& secondary_path)
```

Comment:
Nit: since we are going to store `secondary_path_` as a copy, we can make `secondary_path` of type `std::string`, and do `secondary_path_(std::move(secondary_path))`.